.prettierignore

```diff
@@ -1,4 +1,5 @@
 dist/
 bin/
 public/
-.cache/
\ No newline at end of file
+.cache/
+packages/netlify-cms-backend-github/src/fragmentTypes.js
```

Comment:
`fragmentTypes.js` is auto generated so it should not be formatted

Reply:
If it's auto generated we should pull it out of `src`.

Reply:
Do you mean not to commit it at all and generate it as part of the bootstrap, or just put it in a designated directory?
I think it's useful to have it versioned and consistent between developers, thus I committed it.